engine/src/main/java/org/terasology/rendering/nui/internal/LwjglCanvasRenderer.java

```diff
@@ -113,7 +113,9 @@
 
     public LwjglCanvasRenderer(Context context) {
         // TODO use context to get assets instead of static methods
-        this.textureMat = Assets.getMaterial("engine:UITexture").get();
+        this.textureMat = 
```

Comment:
Trivial but: two lines instead of three here? Can probably combine the first two and still have the `.orElseThrow` flow nicely?